curp-test-utils/src/test_cmd.rs

```diff
@@ -26,10 +26,14 @@ use crate::{META_TABLE, REVISION_TABLE, TEST_TABLE};
 pub(crate) const APPLIED_INDEX_KEY: &str = "applied_index";
 pub(crate) const LAST_REVISION_KEY: &str = "last_revision";
 
+/// Test client id
+pub const TEST_CLIENT_ID: &str = "test_client_id";
+
 static NEXT_ID: Lazy<AtomicU64> = Lazy::new(|| AtomicU64::new(1));
 
-pub fn next_id() -> u64 {
-    NEXT_ID.fetch_add(1, Ordering::SeqCst)
+pub fn next_id() -> String {
+    let seq_num = NEXT_ID.fetch_add(1, Ordering::SeqCst);
```

Comment:
I think `Ordering::Relaxed` would be sufficient.

Reply:
Done